### PR TITLE
Pipes

### PR DESCRIPTION
**Pull request by Notaduck**
CI/CD is working, we can push and travis will build, test and deploy to digital ocean. Hopefully will travis also make a new release on github on every new PR to main.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,86 +1,6 @@
...
-DB_USER=........
-DB_PASSWORD=........
-DB_HOST=..
-ALLOWED_HOSTS=127.0.0.1,localhost,minitwititu.dk
-```
-
-Next step is simply to run the `deploy.sh` script which will tear down any existing containers, then build and run the new ones.
+## How to get deploy
 
+It is fairly easy to deploy thos
```
Comment (emildichmann272) [inline README.md:5]: Not that descriptive, should be fixed in the upcoming commits

File: minitwit_frontend/minitwit_frontend/settings.py [hunk 2]
```diff
@@ -20,13 +20,16 @@
 # See https://docs.djangoproject.com/en/3.1/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.getenv('MINITWIT_SECRET_KEY') or 'vk61e&%q!ggpvhg=nljgy+vg8-tq+5#z%4pesej(wf0^vz%7au'
+SECRET_KEY = os.getenv('SECRET_KEY','vk61e&%q!ggpvhg=nljgy+vg8-tq+5#z%4pesej(wf0^vz%7au')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = bool(os.getenv('DEBUG', 0))
-# DEBUG = bool(int(os.environ.get('DEBUG', 0)))
+DEBUG = bool(int(os.getenv('DEBUG', 1)))
```
Comment (emildichmann272) [inline minitwit_frontend/minitwit_frontend/settings.py]: This should default to 0

**Review by emildichmann272 — APPROVED** [review]
Very minor, to cosmetic details should be altered

File: README.md (hunk 1 above)
Comment (Notaduck) [reply README.md:5]: My mistake :)
But let us get this into dev -> main immediately and then we can update the README.md

File: minitwit_frontend/minitwit_frontend/settings.py (hunk 2 above)
Comment (Notaduck) [reply minitwit_frontend/minitwit_frontend/settings.py]: This has been corrected. The file is outdated now.

**Comment by Notaduck** [comment]
It should be okay to merge this, the one travis job which fails is because it uses org and not com.

**Review by emildichmann272 — APPROVED** [review]
Contains a lot of redundant files, and should be cleaned up before final merge to main

**Pull request merged**